docs/deployment/quick_start.rst

```diff
@@ -41,11 +29,37 @@ First, generate a public and private key pair
     openssl rsa -in deepforge_keys/private_key -pubout > deepforge_keys/public_key
     export TOKEN_KEYS_DIR="$(pwd)/deepforge_keys"
 
+Then create file called ``production-docker-compose.yml`` (for using the keys generated above) with the following
```

Comment:
Putting this file inline is not ideal as it will likely get out-of-date pretty easily. It seems more promising to use something like `docker-compose run` and specify the env variables there.